source/guides/references/protractor-migration-guide.md

```diff
@@ -0,0 +1,568 @@
+  })
+})
+```
+
+## Why Choose Cypress over Protractor?
+
+A common question when teams are considering migrating to a different tool is why choose a different tool when something like Protractor is created specifically for Angular. After all, isn't a tailored tool to the framework better than something more generic like Cypress? 
+
+While this logic holds true in many tool evaluations, end-to-end testing is different because **the value of end-to-end testing resides in ensuring that what the user will see and use on production works**. In other words, whether your application is created using AngularJS, Angular 2, or Angular 9, users will be unaware of this detail. After all, at the end of the day, the user cares primarily about one thing: "Does it work?"
+
+## Benefits of Using Cypress
+
+As many developers can attest to, end-to-end testing is one of those things that they know they *should* do; but often do not. Or if they do run tests, the tests are often flaky and often very expensive due to how long it can take to run. And while there are often ideals of complete code coverage, the realities of business and deadlines often take precedence and the tests are left unwritten, or worse, ignored when errors are being reported because they are not reliable. With Cypress, not only will your tests be reliable, but it provides developers with tools that make e2e testing an asset to development rather than a hindrance.
+
+### Interact with your tests in a browser
+
+When Protractor runs tests, the browser automation launches a browser instance and often runs through tests too fast for the human eye. Without additional configuration, this often leads to a reliance on lengthy terminal errors that can be expensive from a context-switching perspective.
+
+{% video local /img/snippets/cypress-interactive-debugging.mp4 %}
+
+### Faster feedback loops
+
+When it comes to your end-to-end tests, being able to see your tests as they run is critical to allowing you to iterate faster with confidence. Though this is not possible out of the box with Protractor, Cypress tests automatically re-run within the browser whenever they are saved which allows you to iterate faster with confidence.
+
+### Time travel through tests
+
+Rather than trying to decipher errors inside of your terminal, with Cypress you can debug your tests just like you would debug your application normally. In addition, Cypress allows you to see snapshots of your Angular application as the test is executed so you can debug your applications more easily.
+
+### Easily automate screenshots
+
+Another powerful feature of end-to-end testing is the ability to capture screenshots and recording. While possible in both tools, Protractor requires you to manually tap into Node.js and has limited configuration options.
+
+**Before: Protractor**
+
+```javascript
+const fs = require('fs')
+
+describe('Dashboard', () => {
+  it('should render dashboard', () => {
+    i
```

Comment:
@bencodezen Should this "i" be on this line?

Reply:
Not at all. Thanks for catching this!